packages/devtools_app/lib/src/screens/performance/performance_utils.dart

```diff
@@ -87,6 +87,8 @@ void debugTraceEventCallback(VoidCallback callback) {
 
 const preCompileShadersDocsUrl = 'https://docs.flutter.dev/perf/shader';
 
+const impellerWikiUrl = 'https://github.com/flutter/flutter/wiki/Impeller';
```

Comment:
@zanderso / @chinmaygarde are we keeping the wiki as the canonical source of documentation for Impeller?

Reply:
If there is a better link to point users to, let me know and I will change in a follow up PR. Thanks.

Reply:
Yeah the wiki link is fine for now. If/when we shift to flutter.dev, we'll file an issue to update and also put a forwarding link in the wiki.